apps/laboratory/src/components/Ethers/EthersSendCallsWithPaymasterServiceTest.tsx

```diff
@@ -98,7 +112,8 @@ export function EthersSendCallsWithPaymasterServiceTest() {
       )
     }
 
-    return false
+    // Replace with capability check
```

Comment:
Same, is it a todo? Should be replaced with something else?

Reply:
If it requires more polishing or would be handled in another PR etc, would it be better to give more context to devs?

Reply:
Actually a todo yes. Ty

Reply:
Actually there's a typing issue. I'll change this in a different PR 🙏 
Would like to get this merged